tests/NewRelicServiceProviderTest.php

```diff
@@ -46,13 +47,13 @@ public function testRegisterExceptionHandler()
 
         // Define a route that throws an exception
         $app->router->get('/', function() {
-            throw new \Exception();
+            throw new HttpException(550);
```

Comment:
Exceptions are better handled with the latest version of lumen, and regular 500 errors are not spitting out HTML. 

Setting a `550` error (not existing) falls back to the regular error message `Whoops, looks like something went wrong`.

Reply:
This is fine, it's just a test case after all.